inventory/inventory.go

```diff
@@ -26,6 +29,13 @@ type Inventory struct {
 	PackageVulns    []*PackageVuln
 	GenericFindings []*GenericFinding
 	Secrets         []*Secret
+	EmbeddedFSs     []*EmbeddedFS
+}
+
+// EmbeddedFS stores artifacts containing embedded filesystems
```

Comment:
Add a note that this is not proto serialized since it's only used as temporary storage to traverse embedded filesystems during extraction.